services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_egress_config.py

```diff
@@ -168,27 +168,26 @@ def _add_egress_proxy_network(
     service_spec["networks"] = networks
 
 
-def _get_egress_proxy_service_config(
+async def _get_egress_proxy_service_config(
```

Comment:
Why is thisone async ?? pls double check

Reply:
Good catch forgot to revert them